game/src/ecs/items/Item.java

```diff
@@ -68,8 +69,13 @@ public void onDrop(Point position) {
         component.setCollideMethod(
                 (a, b, direction) -> {
                     if (b.equals(ECS.hero)) {
-                        System.out.println("add item to inventory");
-                        ECS.entitiesToRemove.add(droppedItem);
+                        ECS.hero
+                                .getComponent(InventoryComponent.class)
+                                .ifPresent(
+                                        (x) -> {
+                                            ((InventoryComponent) x).addItem(this);
```

Comment:
Sollte das nicht in nem if stehen? Sonst lösch ich das Item auch, wenn der Spieler keinen Platz mehr hat

Reply:
Ja haste, recht wurde behoben.